getting-started-java.md

```diff
@@ -214,16 +211,17 @@ kit.wallet().addEventListener(new AbstractWalletEventListener() {
         // to be double spent, no harm done. Wallet.allowSpendingUnconfirmedTransactions() would have to
         // be called in onSetupCompleted() above. But we don't do that here to demonstrate the more common
         // case of waiting for a block.
-        Futures.addCallback(tx.getConfidence().getDepthFuture(1), new FutureCallback<Transaction>() {
+        Futures.addCallback(tx.getConfidence().getDepthFuture(1), new FutureCallback<TransactionConfidence>() {
             @Override
-            public void onSuccess(Transaction result) {
+            public void onSuccess(TransactionConfidence result) {
                 // "result" here is the same as "tx" above, but we use it anyway for clarity.
                 forwardCoins(result);
             }
 
             @Override
             public void onFailure(Throwable t) {}
         });
+
```

Comment:
I'd rather not add this empty line.

Reply:
Yep, let's remove the empty line. 
Now I'm a bit lost of the next step to do, this is my first time contributing to an open source project :). 
Do I have to change your suggestions in my code and commit again?